### PR TITLE
Added the API_FOR_VMS documentation in the Portal docs.

### LINKED ISSUE #427: Include the API_FOR_VMS documentation in the portal docs.
## Description
Include the API_FOR_VMS documentation in the portal docs so that users can get a fair idea of the feature added.

### Update [Required]
- [ ] [LIST ITEMS]
### Enhancement to Update [Optional]
- [ ] [LIST ITEMS]

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
[2] hours

### PR DESCRIPTION
**Pull request by abhi20161997**
### Description
Added the API_FOR_VMS documentation in the Portal docs  so that users could use it with ease.

Fixes #427 

### Mocks
screenshot from 2018-07-03 12-15-21
screenshot from 2018-07-03 12-16-01
screenshot from 2018-07-03 12-16-38

### Type of Change:

- Documentation

### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have made corresponding changes to the documentation

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: docs/develop/api_for_vms.rst [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+In case of a ``POST`` request, a list containing the details of all meetups
+greater than or equal to the posted date will be returned.
+
+Details of the API
+------------------
+
+1. Use Case : Send event details
+
+2. Endpoint : Volunteers can contribute accordingly
```
Comment (sara-02) [inline docs/develop/api_for_vms.rst]: Endpoint: Is usually the link where the request is sent. `localhost/meetup/api/v1/request_meetup_data/` in this case.  
You can have the `UseCase: Send event details for volunteers to contribute accordingly.` Instead

**Comment by abhi20161997** [comment]
@sara-02 .I've applied the suggested change can you please have a look now ?

**Comment by sara-02** [comment]
@sa7i Can you please merge this

**Comment by Fenn-CS** [comment]
Looks good @abhi20161997 you can just add real examples to it with real turn values to make look better.

File: docs/develop/api_for_vms.rst [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        "Date” : “date after which all meetups are required (yyyy-mm-dd)”
+      }
+
+5. Response Body - Success::
+
+      {
+  “Event Name/Title” : “ --------”
+     “Start Date” : “-----------”
+     “Venue” : “------------------”
+      }
```
Comment (sa7i) [inline docs/develop/api_for_vms.rst]: As seen [here](https://github.com/abhi20161997/portal/blob/26e11e6a9ffbfff04206d76a99aaa5d1b513f31d/docs/develop/api_for_vms.rst), the Response Body - Success and Response Body - Error can be formatted nicely.

File: docs/develop/api_for_vms.rst [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+===========================
+The purpose of building this API was to provide information about the meetups
+in Portal to VMS so that their volunteers could participate in them. Presently,
+there are three fields of meetups which are sent to VMS via the API::
+
+1. Event Name
+2. Start Date
+3. Venue
+
+Users can send in a ``GET`` or a ``POST`` request to the specified `URL <localhost/meetup/api/v1/request_meetup_data/>`_ to access the meetup data.
```
Comment (sa7i) [inline docs/develop/api_for_vms.rst]: On clicking the `URL` it shows a 404 page. 
Remove `URL` and instead replace it with https://localhost/meetup/api/v1/request_meetup_data/

File: docs/develop/api_for_vms.rst [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+In case of a ``GET`` request, a list containing the details of all meetups
+in the ascending order of their dates will be returned.
+
```
Comment (sa7i) [inline docs/develop/api_for_vms.rst]: Provide examples for GET and POST requests.
Each example could include a sample request and the json response from the api

File: docs/develop/api_for_vms.rst [hunk 5]
```diff
@@ -0,0 +1,46 @@
+Api to communicate with VMS
+===========================
+The purpose of building this API was to provide information about the meetups
+in Portal to VMS so that their volunteers could participate in them. Presently,
+there are three fields of meetups which are sent to VMS via the API::
+
+1. Event Name
```
Comment (sa7i) [inline docs/develop/api_for_vms.rst]: Please include the description to the the three fields - Event Name, Start Date and Venue.

**Comment by abhi20161997** [comment]
@sa7i I've implemented the suggested changes. Please can you have a look now ?

File: docs/develop/api_for_vms.rst [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+===========================
+The purpose of building this API was to provide information about the meetups
+in Portal to VMS so that their volunteers could participate in them. Presently,
+there are three fields of meetups which are sent to VMS via the API::
+
+1. Event Name - Title of the event going to be held.
+2. Start Date - The date from which the event would start.
+3. Venue - The location of the event.
+
+Users can send in a ``GET`` or a ``POST`` request to the specified `URL <localhost/meetup/api/v1/request_meetup_data/>`_ to access the meetup data.
```
Comment (sa7i) [inline docs/develop/api_for_vms.rst]: Clicking the URL shows a 404 page.
Remove URL and instead replace it with https://localhost/meetup/api/v1/request_meetup_data/

**Comment by abhi20161997** [comment]
@sa7i Implemented the suggested change.

File: docs/develop/api_for_vms.rst [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+   :align: center
+
+Details of the API
+------------------
+
+1. Use Case : Send event details for volunteers to contribute accordingly.
+
+2. API Method : GET/POST
+
+3. URL Parameters : ``https://localhost/meetup/api/v1/request_meetup_data/``
```
[the remark this answers is not in the document]
Comment (abhi20161997) [reply docs/develop/api_for_vms.rst]: This url is specific for the development branch, once it goes to the production branch, I'll make sure it's ammended.

**Comment by abhi20161997** [comment]
@Vaibhavi24 I've put into effect your suggested changes. Please can you have  a look now ?

File: docs/develop/api_for_vms.rst (hunk 7 above)
[the remark this answers is not in the document]
Comment (abhi20161997) [reply docs/develop/api_for_vms.rst]: This is specific to the development branch. Once it goes on to the production branch, I'll make sure It's changed there.

**Comment by abhi20161997** [comment]
@sa7i  Is it fine now ?

**Pull request merged**